### DISCUSSION TITLE
User Import

### DISCUSSION THREAD

**Discussion by kalebscholes** [logto-io/logto#3817] [Q&A]
Is there a way to import users from an existing system?

**Comment by charIeszhao** [comment]
Currently we don't have out-of-the-box solution for user import, since the situation varies from case to case. Can you talk about your use case, what system are you migrating from?

**Reply by RaffyeMemon** [reply]
I am not sure if a guy started this discussion find the solution but i am in a similar condition.. we are planning to migrate all users from old logto installation to the latest version .. ! 

i am able to query users via api and see the details .. but i am not sure how can push all these users to the new logto installation and even if it will work ?
